lib/utils/check-valid-install.js

```diff
@@ -21,33 +22,47 @@ function checkValidInstall(name) {
 If you are trying to upgrade Ghost LTS to 1.0.0 please see ${chalk.blue.underline('https://docs.ghost.org/v1/docs/migrating-to-ghost-1-0-0')}.
 Otherwise, please run \`ghost ${name}\` again within a valid Ghost installation.`);
 
-        process.exit(1);
+        if (exit) {
+            process.exit(1);
+        }
+
+        throw new Error('UNSUPPORTED');
```

Comment:
would it make more sense to maybe return `true/false/null` instead of throwing errors? might clean up the find-valid-install code that way

Reply:
I think I've done this now 😄